docs/reference/items.md

```diff
@@ -17,8 +17,8 @@ Directus. For the sake of documentation, we'll use a fictional articles collecti
 
 ::: tip Relational Data
 
-Please see [Relational Data](/reference/introduction#relational-data) and [Field Parameters](/reference/query#fields) to
-learn more.
+The items being returned here don't contain related or nested data. To retrieve related data, see
+[Relational Data](/reference/introduction#relational-data) and [Field Parameters](/reference/query#fields).
```

Comment:
Assuming the following simple example collection setup:
```
a
  id
  b - Many to One to 'b'

b
  id
```
If an item from `a` is requested, by default it contains info about the relation, but no nested data about this relation:
```jsonc
{
  "id": 1,
  "b": 1, // primary key of b
}
```

Hence my suggestion:

```suggestion
By default, the item object doesn't contain nested relational data. To retrieve nested data, see [Relational Data](/reference/introduction#relational-data) and [Field Parameters](/reference/query#fields).
```